lang/python/python/Makefile

```diff
@@ -36,7 +36,7 @@ PKG_BUILD_DIR:=$(BUILD_DIR)/Python-$(PKG_VERSION)
 HOST_BUILD_DIR:=$(BUILD_DIR_HOST)/Python-$(PKG_VERSION)
 
 PKG_CONFIG_DEPENDS:= \
-	CONFIG_PACKAGE_python-setuptools CONFIG_PACKAGE_python-pip \
```

Comment:
i would drop this commit entirely ; 
all should work fine without changing anything here

the reason for adding `CONFIG_PACKAGE_python-setuptools` and `CONFIG_PACKAGE_python-pip` here is to re-build Python whenever `python-setuptools` & `python-pip` is selected/deselected in the menuconfig ;

setuptools & pip are installed from the Python tar-ball [and not from an external source]; the reason for this was to reduce work when updating ; only the Python tar-ball needs an update and all 3 get updated, pushing the compat-work [of Python + setuptools + pip] to the Python maintainers